File: HurbAlpha/HurbAlpha/Model/Pagination.swift
//
//  Pagination.swift
//  HurbAlpha
//
//  Created by Julia Rocha on 04/09/19.
//  Copyright © 2019 Julia Rocha. All rights reserved.
//

import Foundation

// MARK: - Declaration

struct Pagination: Codable {
    let count: Int
    let firstPage, nextPage: String
    let previousPage: String?
    let lastPage: String
}

File: app/src/main/java/br/com/rvvaranda/hu/Model/Pagination.kt
package br.com.rvvaranda.hu.Model


import com.google.gson.annotations.SerializedName

data class Pagination(
    @SerializedName("count")
    val count: Int,
    @SerializedName("firstPage")
    val firstPage: String,
    @SerializedName("lastPage")
    val lastPage: String,
    @SerializedName("nextPage")
    val nextPage: String,
    @SerializedName("previousPage")
    val previousPage: Any
)